src/components/inputs/date/ohri-date.component.tsx

```diff
@@ -151,7 +151,9 @@ const OHRIDate: React.FC<OHRIFormFieldProps> = ({ question, onChange, handler })
               className={`${styles.datePickerOverrides} ${isFieldRequiredError ? styles.errorLabel : ''} ${
                 question.disabled || isTrue(question.readonly) ? styles.disabled : ''
               }`}
-              dateFormat={carbonDateformat}>
+              dateFormat={carbonDateformat}
+              maxDate={new Date()} // Set the maximum date to the current date
```

Comment:
Thank you @lucyjemutai for your input. We have forms designed for scheduling appointments, such as setting the next drug pick-up date or selecting the EDD for pregnancy. I propose incorporating a boolean option into the schema. This would allow us to either include the maxDate for a control where we are confident that future dates won't be necessary, or alternatively, provide both the maxDate and minDate parameters within the schema. This approach offers versatility and ensures proper date range constraints are applied for different use cases.

Reply:
@lucyjemutai did you see @donaldkibet's comment?

Reply:
Yes, I saw @donaldkibet's comment.

Reply:
@donaldkibet  - Please clarify the problem statement. If possible, use an example or a use case/user story. Currently, with `minDate` and `maxDate1 in place, the controls are set. What more is needed? Please clarify

`allowFutureDates` should allow a date in the future to be selected. 

`minDate` The minimum date that a user can start picking from.

`maxDate` The maximum date that a user can pick to.

@samuelmale - If you can, please clarify this